pkg/manifest/export_test.go

```diff
@@ -58,6 +59,14 @@ func (p *OS) GetPackageSetChain(d Distro) []rpmmd.PackageSet {
 	return p.getPackageSetChain(d)
 }
 
+func (p *OS) AddFiles(pipeline *osbuild.Pipeline, files []*fsnode.File) {
```

Comment:
(super nitpick) this part did not get renamed but its fine to ignore as its just export_test

Reply:
Woopsie